<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1127</width>
    <height>831</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inventory Management System</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>1101</width>
      <height>561</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Inventory Details</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>60</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Century Gothic&quot;;
font: 14pt &quot;微軟正黑體&quot;;</string>
      </property>
      <property name="text">
       <string>查詢存貨狀態:</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="table">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>110</y>
        <width>1081</width>
        <height>411</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="autoScrollMargin">
       <number>16</number>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>200</number>
      </attribute>
      <column>
       <property name="text">
        <string>PNUMBER</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>PNAME</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>P-SAFETY-STOCK</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="search_btn">
      <property name="geometry">
       <rect>
        <x>950</x>
        <y>60</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;微軟正黑體&quot;;</string>
      </property>
      <property name="text">
       <string>查詢</string>
      </property>
     </widget>
     <widget class="QComboBox" name="table_list">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Century Gothic&quot;;</string>
      </property>
      <item>
       <property name="text">
        <string>PRODUCT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PRODUCT_INVENTORY</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RAW_MATERIAL</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MATERIAL_INVENTORY</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>BOM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>WAREHOUSE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SUPPLIER</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>EMPLOYEE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PURCHASE_ORDER</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="query_list">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>60</y>
        <width>451</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Century Gothic&quot;;
font: 14pt &quot;微軟正黑體&quot;;</string>
      </property>
      <item>
       <property name="text">
        <string>低於安全存貨的產品</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>低於安全存貨的原料</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>需要的原料低於目前存貨的產品</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>需要的原料沒有低於目前存貨的產品</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>若任一原料存量過500，顯示有貨的倉庫</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>若原料存量都沒過500，顯示有貨的倉庫</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>找出目前有幾筆木板的訂單</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>WH04號倉庫目前的總存量(產品+物料)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>存貨量最多的產品</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>存貨量最少的產品</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>所有產品目前的平均存量</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>將至少存有2樣以上產品的倉庫資訊列出來</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_35">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>291</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>選擇要查詢的Table:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>10</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Century Gothic&quot;;
font: 14pt &quot;微軟正黑體&quot;;</string>
      </property>
      <property name="text">
       <string>From MySQL DB:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="search_btn_2">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>10</y>
        <width>391</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;微軟正黑體&quot;;</string>
      </property>
      <property name="text">
       <string>2022manufacture_product_order</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Edit Inventory</string>
     </attribute>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>50</y>
        <width>71</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>名稱</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>0</y>
        <width>71</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>編號</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="name_info">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>60</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Century Gothic&quot;;
color: rgb(173, 66, 164);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>50</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>安全存量</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="safety_stock">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>60</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Century Gothic&quot;;
color: rgb(173, 66, 164);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_info">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>10</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Century Gothic&quot;;
color: rgb(173, 66, 164);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="insert_btn">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>460</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;微軟正黑體&quot;;
font: 16pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Insert</string>
      </property>
     </widget>
     <widget class="QPushButton" name="delete_btn">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>460</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;微軟正黑體&quot;;
font: 16pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QPushButton" name="update_btn">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>460</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;微軟正黑體&quot;;
font: 16pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
     <widget class="QPushButton" name="search_btn_3">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>10</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;微軟正黑體&quot;;</string>
      </property>
      <property name="text">
       <string>查詢</string>
      </property>
     </widget>
     <widget class="QComboBox" name="table_list_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Century Gothic&quot;;</string>
      </property>
      <item>
       <property name="text">
        <string>PRODUCT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RAW_MATERIAL</string>
       </property>
      </item>
     </widget>
     <widget class="QTableWidget" name="table_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>1071</width>
        <height>341</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="autoScrollMargin">
       <number>16</number>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>200</number>
      </attribute>
      <column>
       <property name="text">
        <string>PNUMBER</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>PNAME</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>P-SAFETY-STOCK</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_34">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>291</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>選擇要更改的Table:</string>
      </property>
     </widget>
     <zorder>label_23</zorder>
     <zorder>label_24</zorder>
     <zorder>name_info</zorder>
     <zorder>label_27</zorder>
     <zorder>safety_stock</zorder>
     <zorder>id_info</zorder>
     <zorder>insert_btn</zorder>
     <zorder>delete_btn</zorder>
     <zorder>update_btn</zorder>
     <zorder>search_btn_3</zorder>
     <zorder>table_list_3</zorder>
     <zorder>label_34</zorder>
     <zorder>table_3</zorder>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 22pt &quot;微軟正黑體&quot;;
background-color: rgb(153, 153, 153);</string>
    </property>
    <property name="text">
     <string>存貨管理系統</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>650</y>
      <width>1111</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="title">
     <string>SQL Query</string>
    </property>
    <widget class="QPlainTextEdit" name="query_text">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>401</width>
       <height>91</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_result">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>20</y>
       <width>461</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="query_btn">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>60</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;微軟正黑體&quot;;</string>
     </property>
     <property name="text">
      <string>執行</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="refresh_btn">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>620</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;微軟正黑體&quot;;</string>
    </property>
    <property name="text">
     <string>刷新</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>0</y>
      <width>161</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Century Gothic&quot;;
font: 14pt &quot;微軟正黑體&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../螢幕擷取畫面 2022-12-29 094730.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1127</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
